cmd/vclusterctl/cmd/get/service_cidr.go

```diff
@@ -0,0 +1,78 @@
+package get
+
+import (
+	"fmt"
+
+	"github.com/loft-sh/vcluster/cmd/vclusterctl/flags"
+	"github.com/loft-sh/vcluster/cmd/vclusterctl/log"
+	"github.com/loft-sh/vcluster/pkg/helm/values"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type serviceCIDRCmd struct {
+	Context   string
+	Namespace string
+	log       log.Logger
+}
+
+func getServiceCIDR(globalFlags *flags.GlobalFlags) *cobra.Command {
+	cmd := &serviceCIDRCmd{
+		Context:   globalFlags.Context,
+		Namespace: globalFlags.Namespace,
+		log:       log.GetInstance(),
+	}
+
+	cobraCmd := &cobra.Command{
+		Use:   "service-cidr",
+		Short: "Prints Service CIDR of the cluster",
+		Long: `
+#######################################################
+############### vcluster get service-cidr  ###################
```

Comment:
Please remove the extra `#####` at the end

Reply:
done